packages/commons/logging-execa/src/loggingExeca.ts

```diff
@@ -9,15 +9,17 @@ export declare namespace loggingExeca {
         substitutions?: Record<string, string>;
     }
 
-    export type ReturnValue = ExecaReturnValue;
+    export type Return = Omit<ExecaReturnValue, "stdout" | "stderr"> &
+        // the execa types are incorrect and sometimes stdout and stderr and not defined
+        Partial<Pick<ExecaReturnValue, "stdout" | "stderr">>;
```

Comment:
this was the root cause of https://github.com/fern-api/fern/pull/4675, so updating the types here to prevent it from happening elsewhere